app/ledger.js

```diff
@@ -981,8 +986,12 @@ var stickyP = (publisher) => {
   var result = (siteSetting) && (siteSetting.get('ledgerPayments'))
 
   // NB: legacy clean-up
-  if ((typeof result === 'undefined') && (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined')) {
-    result = synopsis.publishers[publisher].options.stickyP
+  if (typeof result === 'undefined') {
+    if (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined') {
+      result = synopsis.publishers[publisher].options.stickyP
+    } else {
+      result = true
+    }
     appActions.changeSiteSetting(pattern, 'ledgerPayments', result)
   }
   delete synopsis.publishers[publisher].options.stickyP
```

Comment:
yep, I removed last comment with wrong video showing STR. After your update it's working fine.

What I saw is that, if you have a clear session with a synopsis file, publishers you had previously are affected by changing auto-suggest, which should only be applied for new publishers:

STR:
1. remove session-store file
2. keep ledger-synopsis.json (supposing it has publishers)

![gif_auto_include_old_synopsis](https://cloud.githubusercontent.com/assets/4672033/23979914/3a61c918-09db-11e7-96f7-0f2f66353ddc.gif)

I suppose auto-include should only apply for new publishers and not for existing ones? If I'm wrong then PR is good to go ;)